crates/sui-json-rpc/src/transaction_execution_api.rs

```diff
@@ -193,6 +193,12 @@ impl TransactionExecutionApi {
             None
         };
 
+        let raw_effects = if opts.show_raw_effects {
```

Comment:
Should the raw_effects field by an `Option<Vec<u8>>` instead of just `Vec<u8>`?

Reply:
Maybe this isn't important because of how the serialization works, so feel free to ignore if that's the case.